tests/plugins/test_reactor_config.py

```diff
@@ -17,6 +17,7 @@
 import re
```

Comment:
Please refactor the pre_reactor_config and test_reactor_config changes (only) into a separate commit at the start of the series to make it a little easier to review.

Reply:
well it is whole one logical change, and it all depends on each other, as all has to be in place at the same time